packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-sign-in-soc-idp/nodejs/integrationsteps.md

```diff
@@ -0,0 +1,107 @@
+## Integration steps
```

Comment:
code snippets are still referring to the .net sample

Reply:
Yes, I need to work on this one this afternoon.